### ISSUE TITLE
Plugin does not run tests when kotest is included as composite build

### ISSUE THREAD

**Issue by matejdro** [kotest/kotest#5394]
[Gradle's composite builds](https://docs.gradle.org/current/userguide/composite_builds.html) allows for inclusion of a kotest repo directly into application's project, allowing for work on kotest code and immediately running tests in actual application with modified kotest code. This makes editing kotest and making PRs much easier.

However, IntelliJ plugin does not appear to support it. When kotest is included as a composite build, pressing run button will just disable the run button actual test being executed. (and then run button is disabled until IntelliJ / Android Studio is restarted).

Steps to reproduce:

1. Checkout [kotest repo](https://github.com/kotest/kotest) somewhere to your local computer
2. Download and open [KotestDemoApplication.zip](https://github.com/kotest/kotest-intellij-plugin/files/10002809/KotestDemoApplication.zip) in Android Studio (IntelliJ would probably work fine, but so far I've reproduced it in Android Studio)
3. Open settings.gradle
4. Replace `PATH_TO_KOTEST_HERE` with the path to the kotest repo (that you checked out in step 1)
5. Sync project with gradle
6. open `/app/src/test/java/com/example/kotestdemoapplication/ExampleUnitTest.kt`
7. Attempt to run `ExampleUnitTest: Given`

Note that nothing happens. 

* Running just `ExampleUnitTest` file (which would run it as regular JUnit5 test) works fine, so the issue is in the Kotest plugin
* If you comment out `includeBuild` line in the settings.gradle, restart the Android Studio and re-sync the project, running test will work. 

This is with:

```
Android Studio Electric Eel | 2022.1.1 Beta 5
Build #AI-221.6008.13.2211.9237616, built on October 31, 2022
Runtime version: 11.0.15+0-b2043.56-8887301 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 6.0.7-arch1-1
GC: G1 Young Generation, G1 Old Generation


kotest-plugin-intellij (1.2.63-IC-2022.1)
```

**Comment by Kantis** [comment]
I'm unable to reproduce this in IntelliJ (albeit with a different project)

Tested:
1. Adding kotest as a includedBuild
2. Running both a spec and individual tests using gutter icons

I noted some 5-10 seconds of loading happening the first time pressing the gutter icon for the individual test.

**Comment by matejdro** [comment]
Maybe it's specific to Android Studio?

**Comment by Kantis** [comment]
> Maybe it's specific to Android Studio?

Could you give it another shot? I'm thinking maybe Kotest was being built in the background after being included.. or something 🤷🏼  If you're still seeing issues I can setup AS and give it a go

**Comment by matejdro** [comment]
I've tried with

```
IntelliJ IDEA 2022.2.3 (Ultimate Edition)
Build #IU-222.4345.14, built on October 5, 2022
kotest-plugin-intellij (1.2.63-IC-2022.2)
Kotlin: 222-1.7.21-release-272-IJ4167.29
```

And weirdly enough, IntelliJ would not even resolve any of the kotest classes when using composite build:

![image](https://user-images.githubusercontent.com/507922/202648325-d3dc985b-e708-427b-b60a-e26d1fff8914.png)

However, using run configuration that was created before enabling composite build will result in the same issue (pressing on play button just disables the button and does nothing).

Is there a way I can enable more logging for this? I've noticed that plugin does not produce any logs when I press play button, so I can't provide any.
